### ISSUE TITLE
makeup-expander: add configuration option for aria-expanded

### ISSUE THREAD

**Issue by ianmcburnie** [makeup/makeup-js#79]
There are edge cases (e.g. a tourtip) where we don't want the host element to use aria-expanded. We can add a configuration option to use this attribute or not. It should be on by default.

Naming should follow this convention: https://github.com/makeup/makeup-js/blob/master/packages/makeup-listbox/src/index.js#L20 (i.e. `useAriaExpanded: true`)

**Comment by saiponnada** [comment]
No longer an issue. We will be using makeup-expander only for valid cases and that logic will be implemented on components source.
